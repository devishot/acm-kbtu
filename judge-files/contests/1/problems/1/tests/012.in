60 30
29 39
11 18
24 47
48 22
58 39
46 50
26 27
10 45
44 52
58 19
35 16
55 29
40 59
32 57
48 59
35 50
3 9
34 11
11 57
45 39
36 1
17 41
26 33
28 5
10 50
57 21
36 12
59 53
38 4
8 37
