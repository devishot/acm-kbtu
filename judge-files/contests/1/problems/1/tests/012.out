60 58 56 55 54 51 49 48 46 44 52 43 42 40 59 53 38 36 35 34 32 31 30 29 28 26 27 33 25 24 47 23 22 20 19 17 41 16 15 14 13 12 11 18 57 21 10 45 39 50 8 37 7 6 5 4 3 9 2 1 
